drivers/power/adp1050/adp1050.h

```diff
@@ -83,17 +90,21 @@
 #define ADP1050_EXP_MASK			NO_OS_GENMASK(15, 11)
 #define ADP1050_MANT_MASK			NO_OS_GENMASK(10, 0)
 
+#define ADP1051_VDROOP_MAXVAL			0xFF
+
 #define ADP1050_VOUT_EXP_MASK			NO_OS_GENMASK(4, 0)
 #define ADP1050_VOUT_GO_COMMAND			0x06
 #define ADP1050_VOUT_OFF			0x00
 
 #define ADP1050_OUTA_FALLING_EDGE_NEGATIVE_MOD	0x03
 #define ADP1050_OUTB_FALLING_EDGE_NEGATIVE_MOD	0x30
-#define ADP1050_OUTA_ON				0x32
-#define ADP1050_OUTB_ON				0x31
-#define ADP1050_SR1_ON				0x23
-#define ADP1050_SR2_ON				0x13
-#define ADP1050_ALL_PWM_OFF			0x33
+#define ADP1050_OUTA_ON				0x3E
+#define ADP1050_OUTB_ON				0x3D
+#define ADP1051_OUTC_ON				0x3B
+#define ADP1051_OUTD_ON				0x37
+#define ADP1050_SR1_ON				0x2F
+#define ADP1050_SR2_ON				0x1F
```

Comment:
I think there is a more scalable way to do this as well. hint: bit manipulation.